test/com/facebook/buck/jvm/kotlin/testdata/kotlin_library_description/com/example/mixed/Example.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.good
```

Comment:
can you try on a non-facebook computer? ive never gotten buck to compile using buck.